.env.dev

```diff
@@ -1,4 +1,7 @@
 # feature flags
+REACT_APP_FEATURE_ZRX_POLYGON=true
+REACT_APP_FEATURE_POLYGON=true
```

Comment:
let's get these on in `.env.develop` too so it works in ephemeral environments and `develop.shapeshift.com`